packages/action-merge/src/components/MergeWizard/MergeResponse.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react'
 import {ExternalEvents} from 'tocco-util'
-import {FormattedMessage, injectIntl} from 'react-intl'
+import {FormattedMessage} from 'react-intl'
 
-const EntityResponseTable = props => {
-  console.log('props.responseEntites', props.responseEntities)
-  if (!props.responseEntities || props.responseEntities.length === 0) return <div/>
+export const EntityResponseTable = props => {
+  if (!props.responseEntities || props.responseEntities.length === 0) return (<div/>)
```

Comment:
remove brackets around <div/> or add also in showPermissionMessage to be consistent